plugins/plugin-webpack/plugin.js

```diff
@@ -375,8 +363,8 @@ module.exports = function plugin(config, args = {}) {
                   colors: true,
                   all: false,
                   assets: true,
-                }
-          )
+                },
+          ),
```

Comment:
Bah Prettier changes… thanks for updating this

Reply:
I'm sorry for all the unrelated changes :(